assets/app/gitignore

```diff
@@ -1 +1,2 @@
 build
+.last_goflutter_check
```

Comment:
The file is only copied on init, how will we make sure this line is added to the `go/.gitignore` afterwards? See #15, it has the same challenge.

Reply:
https://github.com/go-flutter-desktop/hover/pull/12/commits/0225ab0b4978ab6458b1c16d63db3515d5247961